### ISSUE TITLE
Install issue:  "Building the tests requires a recent version of riscv-none-embed-gcc in path"

### ISSUE THREAD

**Issue by marniedunsmore** [mit-plv/riscv-semantics#29]
I must not have "riscv-none-embed-gcc" in my install path, because when I ran ./install.sh, the make got to "make[1]:  riscv-none-embed-gcc:  Command not found"

A more explicit description about where riscv-none-embed-gcc should exist would be helpful.  What should the path be modified to so that "riscv-none-embed-gcc" can be seen during install?

Thank you.

**Comment by samuelgruetter** [comment]
The README says

> If you do not have the compiler in your path, this command will fail. To install the compiler and add it to your path:
> 
> ```
> $ ./install_riscv_gcc.sh
> $ . setup.sh
> ```

Did you do that? Did that work?

**Comment by marniedunsmore** [comment]
OK, I ran ./install_riscv_gctimc.sh and . setup.sh.

Install got further this time.

It it no longer warning about 'riscv-none-embed-gcc.'
.
But now is stuck at make on the directory '/home/ubuntu/Formal/riscv-semantics-master/softfloat-hs'

After entering this directory, it says "*** No targets specified and no makefile found.   Stop."

Then

Makefile:19:  recipe for target 'riscv-tests' failed

etc.

**Comment by benjaminselfridge** [comment]
Did you do a `git submodule update --init --recursive`?

**Comment by marniedunsmore** [comment]
Yes. The recursion seems to fail for update when it hits the softfloat-hs directory.  The riscv-semantics-master/softfloat-hs directory is empty.
 
Actually, I had this problem when I was trying to install GRIFT as well.

Definitely looks like a problem with the recursive update for softfloat-hs.
